src/types/files.ts

```diff
@@ -1,11 +1,9 @@
-import { AccessibilityRequestDocumentCommonType } from 'types/graphql-global-types';
-
 export type FileUploadForm = {
   file: File | null;
   filename?: string;
   uploadURL?: string;
   documentType: {
-    commonType: AccessibilityRequestDocumentCommonType | null;
+    commonType: null;
```

Comment:
Really no big deal. Just wondering if this is hanging around because the property might be repurposed later, or if there was something still tied to it.

Reply:
Guessing it might be tied to the Intake & TRB Document types still?

https://github.com/CMSgov/easi-app/blob/e3239791f5f266d2a8f6bb604ea9a2decdb90ec9/pkg/graph/schema.graphql#L1567-L1568

https://github.com/CMSgov/easi-app/blob/e3239791f5f266d2a8f6bb604ea9a2decdb90ec9/pkg/graph/schema.graphql#L1633

It _is_ a bit odd, though, that the only type we had referenced in here was for Accessibility Requests, and not also the other two...

Reply:
That exported type (`FileUploadForm`) is used in a couple other places around the code base, but I will double check the specific fields - if we can get rid of them or if that object is different from what was found in graphql above, I will nix it

Reply:
Great find! Thank you both

Reply:
I wasn't too familiar with this type at the time. After Clay's details, and if the type happens to be tied up, I would say feel free to leave this the way it is and FE can get back to it whenever upload code might be revisited.

Reply:
Ok I dug around a bit - that type is tied to methods that are passed to reducers and sagas. I know the goal is to eventually nix that stuff, but for now I will leave as is for safety